test/Npgsql.NativeAotTests/Npgsql.NativeAotTests.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>exe</OutputType>
+    <PublishAot>true</PublishAot>
```

Comment:
Should we also add `<PublishTrimmed>true</PublishTrimmed>`, and use this to test trimming as well? We should probably just get rid of Npgsql.TrimmingTests (this does the same but also adds NativeAOT...)?

Reply:
Let's try and see how it goes...

Reply:
Though from my understanding that shouldn't be necessary

Reply:
Yeah, didn't change anything.

Reply:
You're saying that `<PublishAot>` implicitly turns on `<PublishTrimmed>` too? Interesting...

Reply:
> You're saying that <PublishAot> implicitly turns on <PublishTrimmed> too? Interesting...

Yeah, I think that makes since ILLinker attempts to check every single possible code path and remove what's not used.